app/controllers/employer/base_controller.rb

```diff
@@ -1,9 +1,9 @@
 class Employer::BaseController < ApplicationController
```

Comment:
@jmejia: do we need BaseController if we have our permissions under the services folder (specifying which controller a certain user can access)?